apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -620,16 +619,31 @@ export const TicketInfo = ({ organizationId, form, validations, UploadComponent,
 
 const TICKET_SOURCE_SELECT_STYLE: React.CSSProperties = { width: '100%' }
 const DEFAULT_TICKET_SOURCE_CALL_ID = '779d7bb6-b194-4d2c-a967-1f7321b2787f'
+const MOBILE_TICKET_SOURCE_TYPES = [TICKET_SOURCE_TYPES.MOBILE_APP, TICKET_SOURCE_TYPES.MOBILE_APP_RESIDENT, TICKET_SOURCE_TYPES.MOBILE_APP_STAFF]
```

Comment:
Codewise we do not really support Mobile app in our open-source app.. Would be super cool not to drag it into the platform